pyuavcan/transport/can/media/pythoncan/_pythoncan.py

```diff
@@ -90,25 +129,23 @@ def configure_acceptance_filters(self, configuration: typing.Sequence[_media.Fil
             filters.append(f_dict)
         _logger.info('Acceptance filters activated: %s', ', '.join(map(str, configuration)))
         self._bus.set_filters(filters)
-                        
-    def send_msg(self, msg: can.Message) -> None:
-        self._bus.send(msg, timeout=0.1)    # 0.1s to send CAN packet
 
     async def send_until(self, frames: typing.Iterable[_media.DataFrame], monotonic_deadline: float) -> int:
         num_sent = 0
         for f in frames:
             if self._closed:
                 raise pyuavcan.transport.ResourceClosedError(repr(self))
             self._set_loopback_enabled(f.loopback)
-            # check settings for CAN FD, not implemented yet!
             message = can.Message(arbitration_id=f.identifier, is_extended_id=True, data=f.data, is_fd=self._is_fd)
             if f.loopback == True:
                 with self._loopback_lock:
                     self._loop_frames.append(f)
             try:
-                await self._loop.run_in_executor(self._background_executor, self.send_msg, message)
+                await self._loop.run_in_executor(self._background_executor, lambda: self._bus.send(message, timeout=monotonic_deadline - self._loop.time()))
```

Comment:
The line is too long

```suggestion
                await self._loop.run_in_executor(
                    self._background_executor,
                    lambda: self._bus.send(message, timeout=monotonic_deadline - self._loop.time()),
                )
```